tsconfig.json

```diff
@@ -15,6 +15,6 @@
     "strict": true,
     "noImplicitAny": false,
```

Comment:
I advice you to enable that option. There are 12 errors to be fixed: `pnpm exec tsc --noEmit`

![image](https://github.com/Adamant-im/currencyinfo/assets/25831507/9893de64-cbc7-46cf-ba95-829b6e6e700f)